translations/en.json

```diff
@@ -2750,5 +2750,18 @@
     "import-keypair-steps": "{{account-name}} was derived from your {{keypair-name}} key pair, which has not yet been imported to this device. To transact using this account, you will need to import the {{keypair-name}} key pair first.",
     "not-now": "Not now",
     "share-usage-data": "Share usage data with Status",
-    "value-higher-than-send-amount": "This value is higher than entered amount to send"
+    "value-higher-than-send-amount": "This value is higher than entered amount to send",
+    "help-us-improve-status": "Help us improve Status",
+    "collecting-usage-data": "Collecting usage data helps us improve Status.",
+    "what-we-will-receive":"What we will receive:",
+    "ip-address":"IP address",
+    "universally-unique-identifiers-of-device":"Universally Unique Identifiers of device",
+    "logs-of-actions-withing-the-app":"Logs of actions within the app, including button presses and screen visits",
+    "what-we-not-receive":"What we not receive:",
```

Comment:
There seems to be a typo according to Figma https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=13726-23136&t=OcSHCbXIuyf0kn5Z-0